### ISSUE TITLE
howdy: tests passing, passing options?

### ISSUE THREAD

**Issue by bewest** [watson/mongodown#2]
Howdy,

I have a situation where I already have a mongo connection db and collection instances, and would like to try out `mongodown` as a way to introduce the code to the levelup ecosystem.

I added a few lines to attempt re-using `{conn: <mongojs.connect( )>, collection: 'mycollection'}` as options, but got snagged when I couldn't get tests to pass before or after my changes.  Any advice?  Should I comment out some tests?

**Comment by watson** [comment]
This one slipped through the cracks... The issue seems to be related to MongoDB. The tests parse fine on v2.6, but fails on v2.4. I'll make a fix

**Comment by watson** [comment]
@bewest I finally managed to track down and fix this bug. It turned out that the module didn't work in MongoDB 2.4.x (so the tests broke if running on that version of Mongo of cause). It has now been fixed  in 5390914 and released as v1.0.2 on NPM.

I've also setup a [Travis-CI build matrix](https://travis-ci.org/watson/mongodown) now so that I always test against the different versions of MongoDB (currently 2.4, 2.6 and 2.8)
